src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -1092,6 +1092,47 @@ private Autogenerated.StateItem ToAutogeneratedStateItem(StateTransactionRequest
         }
         #endregion
 
+        #region Configuration API
+        /// <inheritdoc/>
+        public override Task<IReadOnlyList<ConfigurationItem>> GetConfiguration(
```

Comment:
Suggestion: let's change this to `ConfigurationResponse` where it has a property with the items. This way if we add new data to the wire format, we can avoid a bronking change.

Reply:
Done.